modules/data.atmosphere/R/tdm_generate_subdaily_models.R

```diff
@@ -0,0 +1,226 @@
+##' Create statistical models to predict subdaily meteorology from daily means
+##' @family tdm - Temporally Downscale Meteorology
+##' @title gen_subdaily_models
+##' @param outfolder - directory where models will be stored *** storage required varies by size of training dataset, but prepare for >100 GB
+##' @param dat.train_file - train_data file
```

Comment:
This is just `in.path`, but I didn't want to declare it as such because in `tdm_predict_subdaily_met`, we need to read in the `dat.trian_file` again, as `in.path` and `in.prefix` will be used by the coarse data we want to downscale